src/main/java/duke/task/EventTask.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class handles the eventTask.
+ */
+public class EventTask extends Task {
+    private LocalDate date;
+
+    public EventTask(String task) {
+        super(task);
+    }
+
+    String[] divideCommand = task.split(" ");
+
+    @Override
+    public String toString() {
+        String time = "";
+        for (int i = 4; i < divideCommand.length; i++) {
+            if (i == divideCommand.length - 1) {
+                time += divideCommand[i];
+            } else {
+                time += divideCommand[i] + " ";
+            }
+        }
+        date = LocalDate.parse(time);
+        String dateFormat = date.format(DateTimeFormatter.ofPattern("MMM dd yyyy"));
+        String taskRepresent = divideCommand[1] + " " + divideCommand[2]
```

Comment:
Maybe taskDisplayStr or taskDetailsStr might be clearer in showing this string's functionality?